docker-compose.yml

```diff
@@ -41,21 +41,30 @@ services:
         sleep 10;
         npm start;
       "
-  poet-bitcoind:
+  bitcoin-testnet:
```

Comment:
I don't see the need to stand up a testnet docker instance anymore now that we are using regtest. If that is correct, can we either comment this out or remove it? Otherwise when I run `docker up --build` I am starting up a testnet instance and syncing with the testnet blockchain, thus using up bandwidth and storage for no real reason.

Reply:
Good point @geoffturk. I can't think of a situation that I would still want a testnet docker instance locally.

Reply:
Actually, now that I thought about it more, I think we do want a mainnet/testnet instance actually for people to run the node on their machines. Maybe we just need a simpler way of expressing which instance we run instead of running all of them, but that can be tackled in a future PR.

Reply:
That's correct, I'll split it apart in different docker-compose files. We still want to test stuff in testnet if possible, but `docker up` shouldn't cause you to download the whole testnet.